docs/modules/ROOT/pages/references/architecture/control-api-team.adoc

```diff
@@ -0,0 +1,33 @@
+= {controlapi}: Team
+
+TIP: This resource implements the xref:references/functional-requirements/portal.adoc#_feature_manage_teams["Manage Teams"] features.
+
+== Object
+
+.CRD based
+[source,yaml]
+----
+apiVersion: appuio.io/v1
+kind: Team
+metadata:
+  name: myteam1
+  namespace: org-acme-corp <1>
+spec:
+  displayName: My Super Team 1
+  userRefs: <2>
+  - id: bec0d928-2ae2-4cec-94a0-5f72f12b8b39
+  - userName: peter.muster
+status:
+  resolvedUserRefs: <3>
+  - id: bec0d928-2ae2-4cec-94a0-5f72f12b8b39
+    userName: kate.demo
+  - id: 508a9160-977c-4c57-963f-c7b511c4ecc5
+    userName: peter.muster
```

Comment:
Is it now supposed to be `userName` or `username`. I remember we talked about it but do not remember what we condluded and what the reasoning was.